ext/civicrm_admin_ui/ang/afsearchCiviCRMQueues.aff.php

```diff
@@ -0,0 +1,12 @@
+<?php
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  'type' => 'search',
+  'title' => E::ts('CiviCRM Queues'),
+  'icon' => 'fa-list-alt',
+  'server_route' => 'civicrm/queues',
```

Comment:
I'm happy to merge this PR. My only outstanding question is should the path perhaps be:
```suggestion
  'server_route' => 'civicrm/admin/queue',
```
What do you think @eileenmcnaughton ?